_includes/layout/base/html-head.html

```diff
@@ -8,6 +8,11 @@
 <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=no">
 <title>{% capture title %}{% translate title %}{% endcapture %}{% if title != '' %}{{ title }}{% else %}{{ page.title }}{% endif %}</title>
 {% capture metadescription %}{% translate metadescription %}{% endcapture %}{% if metadescription != '' %}<meta name="description" content="{{ metadescription }}">{% endif %}
+
+{% comment %}<!-- load Font Awesome but hide it if JS is disabled -->{% endcomment %}
+<link rel="stylesheet" href="/css/font-awesome-4.4.0/css/font-awesome.min.css">
+<noscript><style>.fa { display: none !important; }</style></noscript>
```

Comment:
@harding I think we could actually avoid this extra line, has it also has the unintended consequence of hiding the icons on browsers with javascript disabled (they display correctly on FF3.5,40 with JS disabled). When we let icons show while NoScript is enabled, the result doesn't look particularly bad to me compared with the other option of just showing no icons (screenshot attached). Also, I have heard that Google doesn't like noscript tags.

![capture du 2015-08-21 15 14 54](https://cloud.githubusercontent.com/assets/3578089/9417491/c413acb4-481a-11e5-9423-9709520974c9.png)